### PR TITLE
djax new bidder adapter 

### PR DESCRIPTION
**Pull request by djaxbidder**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [x] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change

- test parameters for validating bids
```
{
  bidder: 'djax',
  params: {
    publisherId: 1
  }
}
```

Be sure to test the integration with your adserver using the [Hello World](/integrationExamples/gpt/hello_world.html) sample page.

- contact email of the adapter’s maintainer
- [x] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

- A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/pull/1482

### CODE REVIEW

File: modules/djaxBidAdapter.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+            }
+          }]
+  
+        },
+      //video object
+      {
+         code: 'video-ad-slot',
+         mediaTypes: {
+              video: {
+                context: 'instream',
```
[the remark this answers is not in the document]
Comment (djaxbidder) [reply modules/djaxBidAdapter.md]: Hi @jsnellbaker 
If you send the request to bidder adapter URL without parameters (request parameters) , you have the above output. Now it fixed. Now it returns empty json response (if you send the request without parameters)

**Comment by djaxbidder** [comment]
> ?



> Hi @djaxbidder
> 
> Thanks for submitting this new adapter.
> 
> When I tried to test the adapter (for banner), the response I got from the adserver wasn't json code. It seemed like a raw GIF file (based on the response headers and what I saw when debugging the code). I attached a screenshot below of what was in the response field:
> Screen Shot 2019-09-17 at 9 01 51 AM
> 
> Can you please correct this?

File: modules/djaxBidAdapter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (djaxbidder) [reply modules/djaxBidAdapter.md]: Reply For : https://github.com/prebid/Prebid.js/pull/4192#pullrequestreview-289240578
Hi @jsnellbaker
If you send the request to bidder adapter URL without parameters (request parameters) , you have the above output. Now it fixed. Now it returns empty json response (if you send the request without parameters)

**Comment by jsnellbaker** [comment]
@djaxbidder Did you mean to close this PR?

File: modules/djaxBidAdapter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (djaxbidder) [reply modules/djaxBidAdapter.md]: Reply For : https://github.com/prebid/Prebid.js/pull/4192#discussion_r325150782

Hi @jsnellbaker 
This bidder adapter support both instream & outstream video ad.

**Comment by djaxbidder** [comment]
> @djaxbidder Did you mean to close this PR?

By mistaken pull request was closed. i reopen the pull request

File: modules/djaxBidAdapter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (djaxbidder) [reply modules/djaxBidAdapter.md]: Hi @jsnellbaker 
i re-opened my pull request. can you review that?

**Comment by djaxbidder** [comment]
Hi @jsnellbaker 
We have fixed track.php page. Now track.php is working without any issue. Can you check it once

**Comment by jsnellbaker** [comment]
@djaxbidder I tried to retest the track.php call, but I'm getting a CORS error there.  I copied it below for reference.  Can you take a look?

```
Access to XMLHttpRequest at 'https://demo.reviveadservermod.com/headerbidding_adminshare/www/admin/plugins/Prebid/tracking/track.php' from origin 'http://test.localhost:9999' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

**Comment by djaxbidder** [comment]
Hi @jsnellbaker 
We have fixed "CORS site error" issue. can you check it now

**Comment by jsnellbaker** [comment]
@djaxbidder Thanks for the update; LGTM

**Comment by djaxbidder** [comment]
Hi @harpere ,
Is anything pending from our side for new bidder adapter?. And let me know when our bidder adapter will added to prebid.js library?

Thanks

**Pull request merged**